### PR TITLE
[feature] Migration#change_ttl_active

### PR DESCRIPTION
**Pull request by donaldong**
This adds a method allow us to change ttl for all existing records.

### Test Plan
- [x] Added a test case

### CODE REVIEW

File: lib/redcord/migration/ttl.rb [hunk 1]
```diff
@@ -2,10 +2,22 @@
...
   def change_ttl_passive(model)
-    ttl = model.class_variable_get(:@@ttl)
-    model.redis.set("#{model.model_key}:ttl", ttl ? ttl : -1)
+    model.redis.set("#{model.model_key}:ttl", _get_ttl(model))
+  end
+
+  sig { params(model: T.class_of(Redcord::Base)).void }
+  def change_ttl_active(model)
+    change_ttl_passive(model)
+    model.redis.keys("#{model.model_key}:id:*").map do |key|
```
Comment (katyho) [inline lib/redcord/migration/ttl.rb:19]: https://redis.io/commands/keys I'm not sure we should use the keys operation in production 

Alternatively we could do a scan with a pattern match? 
`redis.call('scan', cursor, 'match', 'model_key:id:*')`

Comment (katyho) [reply lib/redcord/migration/ttl.rb:19]: Maybe we can write a lua script which does a scan, and calls expire on each model key returned from scan. We can loop through the cursor on the client side

https://github.com/chanzuckerberg/redcord/blob/kho/vacuum/lib/redcord/server_scripts.rb#L104 this is still very much a work in progress for the vacuum logic, but I was planning on doing something similar here.

**Comment by donaldong** [comment]
use scan each

:mag_right: [Click to see changes since last update](https://github.com/chanzuckerberg/redcord/compare/026f4ce65a69aae84c1af4621c9892b6e3a522c6..3c9545cb429f8699d928b0c81f784113e2e0b7db).

File: lib/redcord/migration/ttl.rb (hunk 1 above)
Comment (donaldong) [reply lib/redcord/migration/ttl.rb]: > lternatively we could do a scan with a pattern match?

great idea! we should avoid blocking as much as possible.

For the same reason, we probably should not use a script here to block less

**Comment by donaldong** [comment]
clearer spec

:mag_right: [Click to see changes since last update](https://github.com/chanzuckerberg/redcord/compare/afbfbafa918edc8394663df0df222a3b69bc1685..856da645f1894185747677c2b026b37e219787ab).

**Review by katyho — APPROVED** [review]

**Pull request merged**